bin/siteleaf

```diff
@@ -151,6 +152,158 @@ def put_theme_assets(site_id)
   end
 end
 
+def build_jekyll_config(site_id)
+  print "Generating configuration file..."
+
+  site = Siteleaf::Site.find(site_id)
+
+  open("export/_config.yml", /^1\.8/.match(RUBY_VERSION) ? 'w' : 'w:UTF-8') do |f|
```

Comment:
I just bumped the Gem to 1.9.3 since HTTParty no longer supports 1.8 either. We had previously supported 1.8 to be compatible with dev using stocky Ruby on older Mac OS, but I think that time has passed. So yes, this is no longer needed.